ooniprobe/View/RunTest/TestRunningViewController.m

```diff
@@ -78,8 +78,7 @@ -(void)setRuntime{
         //We don't want to show -1 seconds before downloading the URL list
         if (self.totalRuntime <= [MAX_RUNTIME_DISABLED intValue])
             return;
-        NSString *time = NSLocalizedFormatString(@"Dashboard.Running.Seconds", [NSString stringWithFormat:@"%d", self.totalRuntime]);
-        [self.timeLabel setText:time];
+        [self.timeLabel setText:[NSString stringWithFormat:@"%dm %ds", self.totalRuntime/60, self.totalRuntime%60]];
```

Comment:
Below you're using `%ld` and here `%d`. I want to make sure this is intentional.